app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -169,6 +185,35 @@ public String convertMarkup(String markup, Context context, boolean isExportInLi
             head += CSS_PRESENTATION_BEAMER;
         }
 
+        // Extract YAML Front Matter
+        if (!enablePresentationBeamer) {
+            if (appSettings.isMarkdownYamlDisplayEnabled() && markup.startsWith("---")) {
+                Map<String, List<String>> yamlFrontMatterMap = Collections.EMPTY_MAP;
+                yamlFrontMatterMap = extractYamlFrontMatter(markup);
+
+                if (!yamlFrontMatterMap.isEmpty()) {
+                    for (Map.Entry<String, List<String>> entry : yamlFrontMatterMap.entrySet()) {
```

Comment:
watch out for Zim Wiki (as it's transpiled to Markdown), do the files work fine?

maybe some additional unit-test(s) that compares input and output could be helpful generally

Reply:
Thanks, I wasn't aware of Zim Wiki. I will have a look.

Reply:
I had a look at some Zim files but could not find any issues. Do you have anything in mind which could lead to problems?

Reply:
no. i dont use zim either.

@fredericjacob